sdk_tests/sdk_integration.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
should we move this file to test_data folder?

Reply:
i'll remove that